packages/yoroi-extension/app/stores/ada/LedgerConnectStore.js

```diff
@@ -65,6 +65,8 @@ import type {
   GetExtendedPublicKeyResponse,
 } from '@cardano-foundation/ledgerjs-hw-app-cardano';
 
+const CONNECTOR_URL = 'https://emurgo.github.io/yoroi-extension-ledger-connect-vnext/catalyst/#/v3.1';
```

Comment:
If using ledgerjs 4.0.0, this should be `https://emurgo.github.io/yoroi-extension-ledger-connect-vnext/multisig/#/v4.1`, but currently the production branch is still using ledgerjs 3.